docs/releaseNotes.yml

```diff
@@ -68,6 +68,14 @@ items:
             link: https://github.com/emissary-ingress/emissary/issues/4134
         docs: https://github.com/emissary-ingress/emissary/issues/4134
 
+      - title: Decrease metric sync frequency
+        type: change
+        body: >-
+          The new delay between two syncs is 30s.
```

Comment:
Small nit: Can you change this to `The new delay between two metrics syncs is now 30s.` so that the corresponding entry in `CHANGELOG.md` makes sense without the context that the title adds? :)